src/cloud/components/RateLimitAlert.tsx

```diff
@@ -104,7 +108,12 @@ export const RateLimitAlert: FC<Props> = ({alertOnly, className, location}) => {
   })
 
   // banner panel for cardinality limit exceeded
-  if (CLOUD && status === 'exceeded' && resources.includes('cardinality')) {
+  if (
+    CLOUD &&
+    !orgIsIOx &&
+    status === 'exceeded' &&
+    resources.includes('cardinality')
+  ) {
     return (
```

Comment:
This might not be necessary - in theory IDPE shouldn't even be giving us an 'exceeded' from the limits endpoint if the org is iox. But I haven't yet traced whether that has been implemented, so this will safeguard against that happening on the ui side.